deploy/csi-operator.yaml

```diff
@@ -861,6 +855,27 @@ roleRef:
   name: openebs-cstor-csi-registrar-role
   apiGroup: rbac.authorization.k8s.io
 
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: openebs-cstor-csi-iscsiadm
+  namespace: kube-system
+data:
+  iscsiadm: |
+    #!/bin/sh
+    if [ -x /host/sbin/iscsiadm ]; then
+      chroot /host /sbin/iscsiadm "$@"
```

Comment:
if PATH env is set, then we don't need to check for each path to see where binary is present (the last else part covers that).

Reply:
There can be different bin paths present based on the distribution, and even in some cases, if user installed and PATH env is not set then there is problem right?

Reply:
question: `PATH` will be set on the host right? Not inside container. Or does the PATH change when we do `chroot`